src/js/utils/common.js

```diff
@@ -91,6 +91,18 @@ export function getTextWidth(text) {
     return Math.ceil(context.measureText(text).width);
 }
 
+export function urlExists(url) {
+    let http = new XMLHttpRequest ();
+
+    http.open('HEAD', url, false);
+    http.send();
+    if (http.status !== 404) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
![image](https://github.com/betaflight/betaflight-configurator/assets/8344830/35556591-88d9-401e-81e0-b88fea28c780)

```suggestion
    return http.status !== 404;
```